packages/stockflux-components/src/styles/button-icon.css

```diff
@@ -12,7 +12,7 @@
     background-image: url('../images/minimize.png');
```

Comment:
I think these images can be replaced by the svgs we have for the title bar. 
#1324 has the examples of the ones for the other windows title bar.

Reply:
Added svgs in and left component file as before